.github/DISCUSSION_TEMPLATE/new-type.yml

```diff
@@ -0,0 +1,60 @@
+title: "Add <suggested new type name>"
+labels: ["New type"]
+body:
+- type: input
+  id: name
+  attributes:
+    label: Type name
+    description: The name of the terms type to add, in Title Case.
+    placeholder: Business Privacy Policy
+  validations:
+    required: true
+- type: textarea
+  id: alternative-type-names
+  attributes:
+    label: Alternative type names
+    description: Known synonyms for the same terms type.
+    placeholder: | 
+      Business Users Privacy Policy
+      Commercial Privacy Policy
+- type: input
+  id: writer
+  attributes:
+    label: Writer
+    description: The author of the document, in most cases the service provider itself.
+    placeholder: service provider
+- type: input
+  id: audience
+  attributes:
+    label: Audience
+    description: The targeted audience whose rights and duties are defined in the associated terms.
+    placeholder: business user
+- type: input
+  id: object
+  attributes:
+    label: Object
+    description: The object of the commitment, i.e. the information or interaction whose handling will be constrained by the associated terms.
+    placeholder: personal data of business users
+- type: textarea
+  id: examples
+  attributes:
+    label: Examples
+    description: |
+      Examples of documents that contain terms of the suggested type, in the form `URL - Title - Service name`, one per line. Provide the URL address at which the document can be accessed; the title of the document provided at this URL; and the name of the service.
+      e.g. `https://www.partners.skyscanner.net/privacy-policy - Corporate Business Contact Privacy Notice - Skyscanner`
```

Comment:
The full example is given as a placeholder, I don't see the point of repeating it here, we don't do it for other fields 🙂 
```suggestion
```